CHANGELOG.md

```diff
@@ -31,8 +32,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Updated ExtData so that if files are missing in a sequence the last value will be perisisted if one has not chosen `exact` option
-- Changed MAPL_ESMFRegridder to require the dstMaskValues to be added as grid attribute to use fixed masking, fixes UFS issue
```

Comment:
@bena-nasa I don't think you meant to remove the line about `MAPL_ESMFRegridder` right? Just move the first line up?